locales/templates/app.pot

```diff
@@ -204,7 +200,11 @@ msgid "Delete screenshot?"
 msgstr ""
 
 msgctxt "dialogue_delete_content_cantundo"
-msgid "Are you sure? This action cannot be undone."
+msgid "Are you sure? This action will permanently delete all of its screenshots from the phone."
```

Comment:
What is "its" referring to? It sounds a bit awkward to me. Maybe use rather "your" or "these" or "the" - or drop "of its" altogether.

Reply:
This copy is under the string "delete collection?" Therefore, the "its screenshots" explicitly indicate the screenshots in the collection which users want to delete. If we use "your, "these" or simply remove "its," I am afraid it might be a bit ambiguous since users might think this action will delete all screenshots, even those outside of the collection. You could find some similar examples like attached. 

However, I'll update the copy to be more specific like "Are you sure? Deleting this collection will permanently remove all of its screenshots from your phone."

<img width="330" alt="image" src="https://user-images.githubusercontent.com/18019946/46394176-dca9e900-c71a-11e8-9fe4-7814cbed97bb.png">
<img width="272" alt="image" src="https://user-images.githubusercontent.com/18019946/46394300-4f1ac900-c71b-11e8-9461-e4f8d56c395d.png">

Reply:
Thanks for clarifying. We're going to need a localization comment though to explain what "its" refers to, otherwise it is unclear and can't be properly localized (add quick note what "its" refers to and what's going on)